_episodes/30.md

```diff
@@ -0,0 +1,278 @@
+both live and historic data and merge them. This enables reading from live data
+feeding into their stateful data stores, and reads across historic data stores
+to produce data in the form needed to support Talkdesk products.
+
+<p align="center">
+<img align="center" width="90%" height="100%" src="/assets/episode/30/talkdesk-architecture-2.0.png"/><br/>
+</p>
+
+Trino uses also are used to hide the complexity of the data platform and allows
+merging data across mulitple relational and object stores.
+
+<p align="center">
+<img align="center" width="60%" height="100%" src="/assets/episode/30/talkdesk-architecture-2.0-external.png"/><br/>
+</p>
+
+### Why dbt?
+
+In [episode 21](/episodes/21.html) we discussed using dbt + Trino in detail. As
+we mentioned there:
+
+> dbt is a transformation workflow tool that lets teams quickly and 
+> collaboratively deploy analytics code, following software engineering best 
+> practices like modularity, CI/CD, testing, and documentation. It enables 
+> anyone who knows SQL to build production-grade data pipelines. 
+
+You can achieve modularity and repeatable testable units of processing by 
+defining various models and definitions to the data pipelines. For example:
+
+<p align="center">
+<img align="center" width="50%" height="100%" src="/assets/episode/30/dbt-definition.png"/><br/>
+</p>
+
+Using the definitions above, Talkdesk engineers were able to consolidate all
+these tasks into a much more simplified graph of operations. 
+
+<p align="center">
+<img align="center" width="75%" height="100%" src="/assets/episode/30/dbt-results.png"/><br/>
+</p>
+
+### Why data mesh?
+
+While a lot of focus has gone into the technology aspects of data mesh, there is
+also a lot to be said about the implications on the data team and sociopolitical
+policies that come with data mesh. Talkdesk also made structural changes to 
+their team to improve their data mesh strategy.
+
+<p align="center">
+<img align="center" width="50%" height="100%" src="/assets/episode/30/talkdesk-data-team.png"/><br/>
+</p>
+
+### How will Data Mesh affect the everyday life of data engineers?
+
+There is a real fear that comes around when management changes business 
+policies. It can be hard to tell how these policies will trickle down and affect
+the engineer's every day work life. In general engineer's are going to become 
+more entrenched in different domains rather than trying to manage all domains
+under one architecture. Data engineers will be distributed to product teams and 
+specialize in the domain's data models and use of the self-service platform to 
+integrate across other teams.
+
+### Comparing microservices based applications to the data mesh
+
+When we think of a functional system for deploying and managing microservices 
+based applications, there are several features that we’ve come to expect from 
+the platform.
+
+Let’s transform each of these to data mesh platform particular capabilities:
```

Comment:
weird sentence.. reword and add empty line after

Reply:
I've changed it to this, I think this will clear up the following sentences: 

> It is very easy to compare the features of microservices-based 
applications to features of a data mesh

All the following bullet points were summaries of [this blog](https://blog.starburst.io/data-mesh-a-software-engineers-perspective) from Piotr. Do you suggest that we just remove these and link to the blog instead? May be better.

Reply:
Yeah .. maybe thats better